src/openforms/submissions/models/submission_report.py

```diff
@@ -65,15 +65,17 @@ def generate_submission_report_pdf(self) -> str:
         :return: string with the HTML used for the PDF generation, so that contents
           can be tested.
         """
-        form = self.submission.form
-        html_report, pdf_report = render_to_pdf(
-            "report/submission_report.html",
-            context={"report": Report(self.submission)},
-        )
-        self.content = ContentFile(
-            content=pdf_report,
-            name=f"{form.name}.pdf",  # Takes care of replacing spaces with underscores
-        )
+        with override(self.submission.language_code):
+            form = self.submission.form
+            # Switch to submission lang TODO
```

Comment:
what's this TODO about?

Reply:
That was my breadcrumb that the little birdies forgot to eat. :grimacing: :bird: